src/test/tests/applicabilityScan.test.ts

```diff
@@ -114,125 +127,229 @@ describe('Contextual Analysis Scan Tests', () => {
             expectedEvidence: []
         }
     ].forEach(test => {
-        it('Generate response - ' + test.name, () => {
-            let response: ApplicabilityScanResponse = getDummyRunner().generateResponse(getAnalyzerScanRun(test.file));
-            assert.isDefined(response);
+        describe('Generate response - ' + test.name, () => {
```

Comment:
The `describe` should not be in a foreach loop, as it will be generated for each test case.
The `describe` is a more general thing.